src/restic/node.go

```diff
@@ -493,12 +559,35 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 	case "fifo":
 	case "socket":
 	default:
-		err = errors.Errorf("invalid node type %q", node.Type)
+		return errors.Errorf("invalid node type %q", node.Type)
+	}
+
+	if err = node.fillExtendedAttributes(path); err != nil {
+		return err
 	}
 
 	return err
 }
 
+func (node *Node) fillExtendedAttributes(path string) error {
+	if node.Type == "symlink" && node.LinkTarget == "invalid" {
```

Comment:
I can create softlinks with acls (debian jessie):
```
touch /tmp/test

ln -s /tmp/test /tmp/softlink-with-acl

setfacl /tmp/softlink-with-acl

getfacl /tmp/softlink-with-acl
Removing leading '/' from absolute path names
# file: tmp/softlink-with-acl
# owner: root
# group: root
user::rw-
group::r--
other::r--

setfacl -m u:nobody:r /tmp/softlink-with-acl

getfacl /tmp/softlink-with-acl
getfacl: Removing leading '/' from absolute path names
# file: tmp/softlink-with-acl
# owner: root
# group: root
user::rw-
user:nobody:r--
group::r--
mask::r--
other::r--

setfattr -n user.comment -v "this is a comment" /tmp/softlink-with-acl
getfattr -d /tmp/softlink-with-acl
getfattr: Removing leading '/' from absolute path names
# file: tmp/softlink-with-acl
user.comment="this is a comment"
```

So you can set acls and extended attributes on softlinks.
I do not know if it is a bug of the xattr lib I used (the other library has problems on OS X, that's why I used this one). Perhaps the syscall returns an error if the link can not followed.